testsuite/standalone/src/test/java/org/wildfly/core/test/standalone/mgmt/ReadResourceTestCase.java

```diff
@@ -0,0 +1,34 @@
+package org.wildfly.core.test.standalone.mgmt;
```

Comment:
It misses a license header, I'll add it after merging it. Maybe we can have a GitHub action to check this